app/assets/javascripts/actions/projectBoard.js

```diff
@@ -31,6 +53,10 @@ export const expandStoryIfNeeded = (dispatch, getHash) => {
   }
 }
 
+export const sendErrorNotificationIfNeeded = (dispatch, code, condition) => {
+  if (!condition) dispatch(sendErrorNotification(code, { custom: true }));
```

Comment:
I think we should invert this condition, it's not very intuitive to check if the condition was _not_ met in an action called `sendErrorNotificationIfNeeded`. Can we remove the `!` from here and add it to where the function is called?

Reply:
Sure!